src/system/SystemLayer.cpp

```diff
@@ -67,6 +68,24 @@
 namespace chip {
 namespace System {
 
+namespace {
+
+Timer::Epoch GetTimerEpoch(const Callback::Cancelable * timer)
```

Comment:
are timerecho changes supposed to be part of this PR?

Reply:
Yes because this the epoch is stored in the same space..